### PR TITLE
[Gorush] Add Dockerfile for Gorush

### PR DESCRIPTION
**Pull request by emiksk**

### CODE REVIEW

File: gorush/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Gorush container
+
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.12-bionic AS build
```
Comment (zoetrope) [inline gorush/Dockerfile]: Currently, we are using the latest golang version 1.13.
Could you update it?

Comment (emiksk) [reply gorush/Dockerfile]: Thank you for your review.
I'll update it.

Comment (emiksk) [reply gorush/Dockerfile]: Done

**Comment by emiksk** [comment]
@zoetrope 
I updated the version for golang in Dockerfile.

**Comment by aoi1** [comment]
LGTM

**Comment by emiksk** [comment]
@zoetrope 
Thank you for your review!
Could you merge this if this is acceptable for you?

**Comment by emiksk** [comment]
Thanks!

**Pull request merged**